packages/esm-patient-chart-app/src/patient-chart/action-menu/action-menu.scss

```
@@ -98,6 +98,18 @@ $actionPanelOffset: 3rem;
   }
 }
 
+:global(.omrs-breakpoint-lt-desktop) {
+  .hiddenSideRail {
+   display: none;
+  }
+}
+:global(.omrs-breakpoint-lt-desktop) {
+  .showSideRail {
+   display: flex;
+  }
+}
```

```suggestion
:global(.omrs-breakpoint-lt-desktop) {
  .hiddenSideRail {
   display: none;
  }
  .showSideRail {
   display: flex;
  }
}
```

The 2 classes can be clubbed into a single class bracket

---

@vasharma05 thanks, though it was merged already